Makefile

```diff
@@ -127,7 +127,7 @@ gas: $(GAS)
 # convenience targets
 all: components tethers isos vic-machine vic-ui
 tools: $(GOIMPORTS) $(GVT) $(GOLINT) $(SWAGGER) $(GAS) goversion
-check: goversion goimports govet golint copyright whitespace gas
+check: goversion goimports govet lintcode copyright whitespace gas
```

Comment:
thanks. original golint seemed too confusing.
We might need to get some focus on Makefile and clean it up. It takes so long to run even trivial things. Especially if it is a mounted network disk. As well as weird overrides.